### ISSUE TITLE
Remove large io.ascii files from master branch

### ISSUE THREAD

**Issue by astrofrog** [astropy/astropy-benchmarks#23]
It would be better to generate these on-the-fly

**Comment by bsipocz** [comment]
so these are the reason cloning the repo takes forever?

**Comment by MSeifert04** [comment]
If they are really problematically big we might need to rewrite history to avoid keeping them as part of the clone.

**Comment by astrofrog** [comment]
I think the biggest part of the repo is actually the results branch - it has tens of thousands of files.

**Comment by pllim** [comment]
Yeah! Let's squash `master` into 1 commit! `results` branch won't be affected and should not be fetched blindly.

**Comment by bsipocz** [comment]
😱 

Actually is it's only the small files, then cloning only master would solve the issue anyway. And while it indeed takes a while, I can very well live with the long cloning time, we don't do it in CI so local times doesn't really matter on the long run.

**Comment by pllim** [comment]
Results used to be in `master` until @astrofrog moved them to `results`, so that might be the reason why it is slow too? (I still don't understand the `git` clone magic completely.)
